mysql-test/suite/secondary_engine/t/query_preparation_debug.test

```diff
@@ -126,32 +126,7 @@ SET DEBUG = "+d,secondary_engine_mock_compare_cost_error";
 # The injected error causes fallback to the primary engine.
 EXPLAIN SELECT * FROM t1 JOIN t2 ON t1.x = t2.y LEFT JOIN t3 ON t2.x = t3.y;
 SET DEBUG = "-d,secondary_engine_mock_compare_cost_error";
-<<<<<<< HEAD
 SET DEBUG_SYNC = "reset";
-||||||| merged common ancestors
-=======
-
-let $query =
```

Comment:
Is this fragment from upstream removed deliberately?

Reply:
Good catch.